src/main/java/application/GetRelevantConversations.java

```diff
@@ -49,24 +44,22 @@ public GetRelevantConversationsRequest getRequestModel(){
     /**
      * Accepts a request.
      * @param request   a request stored as a RequestModel
+     * TODO: change request() based on the code in 'CliController'.
      */
     @Override
     public void request(RequestModel request) {
-
         GetRelevantConversationsRequest grc_request = (GetRelevantConversationsRequest) request;
 
-        conversationQueue = new ConversationQueue(user.getLocation(), grc_request.locationRadius,
-                user.getInterests());
+        conversationQueue = new ConversationQueue(grc_request.location, grc_request.locationRadius,
```

Comment:
java variables shouldn't have underscores like in python- grcRequest would be more in line with style guide.

Reply:
changed grc_request to grcRequest.